### PR TITLE
SDK v1.2.0

### PR DESCRIPTION
**Pull request by MilosRandelovic**
- New config params: pauseTracking, useCookies
- Fixed a bug with setting cookie time
- Fixed a bug when 401 from the server (if the origin was missing) would not result in an error being thrown
- Added tests & GitHub CI Workflow to run them
- Updated README, package.json and .npmrc - ready for npm publish

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,45 +1,66 @@
...
 ```ts
-import { EventTrackingConfig } from "@wunderbar-network/mini-digital-sdk-js";
+import { EventTrackingConfig } from "@wunderbar-network/mini-digital-sdk";
 
 // Override the default value
-EventTrackingConfig.miniDigitalUrl = "https://test.api.mini.digital/";
+EventTrackingConfig.miniDigitalUrl = "http://localhost:1234/";
 ```
+
+You can also choose to _pause_ sending events (e.g. in your test suites), or not use cookies (to store anonymous tracking IDs):
```
Comment (DanHenton) [inline README.md:56]: 👏

File: README.md [hunk 2]
```diff
@@ -1,45 +1,66 @@
...
-If you wish to override the default Mini Digital URL (to point to e.g. a testing instance), you can do so by overriding the config namespace variable:
+By default, the config is set to send events to the production Mini Digital endpoint. If you wish to override the default Mini Digital URL (to point to e.g. a testing instance), you can do so by overriding the config namespace variable:
 
 ```ts
-import { EventTrackingConfig } from "@wunderbar-network/mini-digital-sdk-js";
+import { EventTrackingConfig } from "@wunderbar-network/mini-digital-sdk";
 
 // Override the default value
-EventTrackingConfig.miniDigitalUrl = "https://test.api.mini.digital/";
+EventTrackingConfig.miniDigitalUrl = "http://localhost:1234/";
```
Comment (DanHenton) [inline README.md:53]: Maybe this should be the default sink port?

**Review by DanHenton — APPROVED** [review]
Awesome work! ✅

File: README.md (hunk 2 above)
Comment (MilosRandelovic) [reply README.md]: Have changed!

**Pull request merged**